web/src/test/java/org/cbioportal/genome_nexus/web/AnnotationIntegrationTest.java

```diff
@@ -35,8 +36,9 @@
 
     private final static String BASE_URL_GENOMIC_LOCATION = "http://localhost:38896/annotation/genomic/";
 
-    private final static String ALL_ENSEMBL_FIELDS = "fields=annotation_summary,nucleotide_context";
-    private final static String MY_VARIANT_INFO_FIELD = "fields=my_variant_info";
+    private final static String ALL_ENSEMBL_FIELDS = "fields=" + AnnotationField.ANNOTATION_SUMMARY.getValue()
+        + "," + AnnotationField.NUCLEOTIDE_CONTEXT.getValue();
+    private final static String MY_VARIANT_INFO_FIELD = "fields=" + AnnotationField.MY_VARIANT_INFO.getValue();
```

Comment:
I think it is better if we do not modify these string values for integration test. Let's keep it as is.

Reply:
We don't need to keep these as string constants, but keeping them as constants ensures that we don't accidentally break backward compatibility. I think it's a good thing to see a failing test after we make a potentially breaking change. If the change is intended we can then manually update the tests.